### DISCUSSION TITLE
The highlighted mark is not displayed correctly

### DISCUSSION THREAD

**Discussion by Gsonovb** [requarks/wiki#5824] [Error / Bug Report] [answered]
HI

I have a problem

In the editor "Highlight syntax (==)" can be seen in the preview,
![image](https://user-images.githubusercontent.com/3164300/200460626-40e3d5c6-c65f-49c1-9932-4d2a3cf2519c.png)

But in the publishing page, the display is incorrect.
![image](https://user-images.githubusercontent.com/3164300/200460691-f14ec4e5-f4b1-4332-b03a-998ddbeb58ca.png)

**Comment by NGPixel** [comment, marked as answer]
The mark tag you're using is not officially supported, as it's very uncommon.

I suggest using inline code instead:

```markdown
something `x`, `y`, `z` something
```

**Reply by jeffshead** [reply]
Thanks for the suggestion but using inline code markdown for emphasizing regular text is not ideal. It will make your page layout inconsistent and confusing since you will be using the same background color for code and for highlighting regular text.

Is there no other way to accomplish highlighting regular text with a different color background using markdown?

**Reply by RogueSnipa** [reply]
@jeffshead 
In such cases, I use the "mark" tag
```html
I need to highlight these <mark>very important words</mark>.
```
